src/3box.js

```diff
@@ -389,17 +386,20 @@ class Box extends BoxApi {
     let proof = await this._readAddressLink(address)
 
     if (!proof) {
-      if (!this._provider.is3idProvider) {
-        try {
+      try {
+        if (!this._provider.is3idProvider) {
           proof = await createLink(this._3id.DID, address, this._provider)
-        } catch (e) {
-          throw new Error('Link consent message must be signed before adding data, to link address to store', e)
-        }
-        try {
-          await this._writeAddressLink(proof)
-        } catch (err) {
-          throw new Error('An error occured while publishing link:', err)
+        } else if (this._provider.threeIdConnect && this._provider.migration) {
+          // during migration, need to link "managment" address, for account which derived, rather than creating general link proofs
+          proof = await this._3id.linkManagementAddress()
```

Comment:
flow just easier, linkManageAddress still holds for creating "new" legacy account during partial migration, and easier to request provider to create link when needed in 3boxjs still (and rpc call available with linkManagementAddress), vs waiting for link (plus would still need some call to request link be created). but yeah comment confusing, since still same proofs, just the req/res and rpc flow related